server/protocols/amnezia.py

```
@@ -0,0 +1,59 @@
+
+    async def is_running(self) -> bool:
+        status = await get_core_status(self.PROTOCOL_ID)
+        return status.get("status") == "running"
+
+    async def add_client(self, username: str, client_data: dict) -> dict:
+        return {"username": username, "status": "ready"}
+
+    async def remove_client(self, username: str, protocol_data: dict):
+        return None
+
+    async def get_client_config(self, username: str, server_ip: str, protocol_data: dict, config_id: str = None) -> dict:
```

Ruff (0.15.2)

[warning] Unused method argument: `protocol_data`

(ARG002)